### DISCUSSION TITLE
[tsml_ui] and CSS colored text

### DISCUSSION THREAD

**Discussion by dougie-wougie** [code4recovery/12-step-meeting-list#1175] [Q&A]
I am using [tsml_ui] and want to change the small text "Closed" to the color red. I tried a few things on my site CSS and it does not reflect any changes. Is it possible and if so, how do I do it? 

d

**Comment by joshreisner** [comment]
to confirm what you mean by the small text "closed" - is that like the gray text "women" in this screenshot?

<img width="551" alt="Screenshot 2023-07-27 at 6 39 41 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/f39cbb74-ec83-4bfe-91c8-9bf10a415a7f">

what css did you try?

**Reply by joshreisner** [reply]
@dougie-wougie ah ok! internally we call those "flags" - do you have other flags set? it's not currently possible to style only one value, although if you would like that it seems easy enough to add

```
#tsml-ui table small {
  color: tomato !important;
}
```

**Reply by dougie-wougie** [reply]
Yes, that changes them all. Cool. However, I only want "Closed" meeting to be colored.

**Reply by joshreisner** [reply]
ok created an issue https://github.com/code4recovery/tsml-ui/issues/320

**Reply by joshreisner** [reply]
to be clear if a closed meeting has other flags on it eg `Closed` and `Women` you only want the `Closed` and not the `Women` to be styled

**Reply by dougie-wougie** [reply]
My thinking is: Closed = Red, Men = Blue, Women = Pink. But, yes each flag would be or could be colored. But, it is eye candy only, so if that is too much work. I can back away from the idea.

**Reply by joshreisner** [reply]
ok, yeah, to be honest as a designer i would not recommend this - it sounds noisy and i'd be a little concerned about the mood it sets on the list. 

from an application standpoint though it seems like you should be able to apply styles to individual flags, so i'll work on it.

perhaps a middle ground for now might be to remove "Open" as a flag if it's also set, because that would increase the prominence of the other flags. you could increase them further still by making them bold:

```
#tsml-ui table small {
  font-weight: bold;
  color: black !important;
}
```

**Reply by dougie-wougie** [reply]
Thanks. I was thinking of backing off. If I have "Closed" meetings as red, folks may assume it is no longer a meeting happening. That is not what I want people to think. It does imply this meeting type is more important than that one and so on.

D

**Comment by dougie-wougie** [comment]
Yes, that is it. I am using three CSS for testing only. I was trying to get anything to work. 2 of the 3 are just for seeing if I am doing it correctly. 

td[data-sort="Closed"] > small {
  	color: red !important;
}

body.tsml-meetings {
  background-color: tomato;
}

.tsml-widget-upcoming h2 {
  font-size: 16px;
}

Thanks... 

D
